packages/utils/scaffold.ts

```diff
@@ -74,14 +76,16 @@ export default function runTransform(transformConfig: ITransformConfig, action:
 					configurationName = "webpack." + config.configName + ".js";
 				}
 
-				const outputPath: string = initActionNotDefined
-					? transformConfig.configPath
-					: path.join(process.cwd(), configurationName);
-				const source: string = ast.toSource({
-					quote: "single",
+				pkgDir(process.cwd()).then((projectRoot) => {
```

Comment:
Using `process.env.PWD` instead of `process.cwd()` in deleted code will also fix the issue IMO.

Reply:
You mean fix the CI or the issue? 🤔

Reply:
issue

Reply:
Ah I just tested and it prints the current dir for me and not the project root dir🤔

Reply:
What is your current dir? This `process.env.PWD` remains constant throughout a process with where it started. And I am assuming that your running `webpack-cli init` or any other of scaffolding packages in project root directory.

Reply:
for instance - 

Project root is `test`

cur dir is `test/a/b`

I run `webpack-cli init`

`webpack.config.js` is generated at the root rather than `/a/b` path

cc @evenstensberg is this what we want?

Reply:
Makes sense, will make changes 👍